renderdoc/core/core.cpp

```diff
@@ -573,6 +553,31 @@ void RenderDoc::Tick()
   prev_cap = cur_cap;
 }
 
+void RenderDoc::ChangeContext()
```

Comment:
Similar to the discussion about the user-facing name, `ChangeContext` is not a good name for most APIs that aren't GL, and is actually misleading in the case of GL. Instead internally this should be `CycleActiveWindow` as it's changing the value of `m_ActiveWindow` by cycling through `m_WindowFrameCapturers`.